multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -48,6 +48,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("Glory Scans", "https://gloryscans.fr", "fr"),
         SingleLang("Gremory Mangas", "https://gremorymangas.com", "es"),
         SingleLang("Hanuman Scan", "https://hanumanscan.com", "en"),
+        SingleLang("Hentai20", "https://hentai20.io", "en", isNsfw = true, overrideVersionCode = 4),
```

Comment:
```suggestion
       SingleLang("Hentai20", "https://hentai20.io", "en", isNsfw = true, overrideVersionCode = 9),
```
This wouldn't be enough to trigger an extension update in-app.
- Madara's `baseVersionCode` is 33, `overrideVersionCode` is 3, so the version would be 36, which is currently the version on the repo: https://keiyoushi.github.io/extensions/#en.hentai20
- MangaThemesia's `baseVersionCode` is 28, so an `overrideVersionCode` of 4 would make the final version 32, which is a downgrade.

Reply:
Oh.... I understand! I will make new one then!